pandas/core/arrays/_mixins.py

```diff
@@ -44,6 +44,10 @@ def _box_func(self, x):
         """
         return x
 
+    def _validate_insert_value(self, value):
```

Comment:
should we just make this class an https://docs.python.org/3/library/abc.html then these aren't even needed

Reply:
my hope is that we'll smooth out the small differences between the various `_validate_foo_value` methods (ATM trying to sort out Categorical's many special cases) and just have `_validate_setitemlike_value` or something like that